packages/oui-progress-tracker/_mixins.less

```diff
@@ -81,20 +86,37 @@
       }
     }
 
-    &__step_complete {
-      cursor: pointer;
-      pointer-events: initial;
+    &__label {
+      position: relative;
+      top: rem-calc(40);
+      min-width: @label-min-width;
+      max-width: @label-max-width;
+    }
 
+    &__connector {
+      position: relative;
+      width: 100%;
```

Comment:
(optional) Since `.oui-progress-tracker` use `flex`, you could go with `flex: 1;` instead of using the width :)

Reply:
The entire class has been removed after re-structuring and clean up of the .less code